crates/transaction-pool/src/pool/pending.rs

```diff
@@ -243,6 +246,23 @@ impl<T: TransactionOrdering> PendingPool<T> {
         removed
     }
 
+    /// Updates the independent transaction and independent descendants set, assuming the given
+    /// transaction is being _added_ to the pool.
+    fn update_independents(&mut self, tx: &PendingTransaction<T>, tx_id: &TransactionId) {
```

Comment:
we should find a better name for this

Reply:
changed to `update_independents_and_highest_nonces`